sdks/python/apache_beam/transforms/util.py

```diff
@@ -221,6 +220,9 @@ def __init__(self,
     self._clock = clock
     self._data = []
     self._ignore_next_timing = False
+
+    # Importing here to prevent a circular dependency.
+    from apache_beam.metrics import Metrics
```

Comment:
Metrics should not depend on transform.util. Fix the circular dependency in Metrics (if needed).